src/main/java/uk/ac/ebi/eva/contigalias/dus/AssemblyReportReader.java

```diff
@@ -75,10 +75,10 @@ private void parseReport() throws IOException, NullPointerException {
                 parseAssemblyData(line);
             } else if (!line.startsWith("#")) {
                 String[] columns = line.split("\t", -1);
-                if (columns.length >= 6) {
-                    if (columns[3].equals("Chromosome")) {
+                if (columns.length >= 6 && columns[5].equals("=")) {
+                    if (columns[3].equals("Chromosome") && columns[1].equals("assembled-molecule")) {
                         parseChromosomeLine(columns);
-                    } else if (isScaffoldsEnabled && columns[1].equals("unplaced-scaffold")) {
+                    } else if (isScaffoldsEnabled) {
```

Comment:
I saw [here](https://github.com/EBIvariation/contig-alias/blob/bfa825a5b7a975ad623c99ec40e24b8e23322937/src/main/java/uk/ac/ebi/eva/contigalias/dus/AssemblyReportReaderFactory.java#L28) that by default `isScaffoldsEnabled ` is false.
Should we set it to true or is it just a deployment configuration to make ?

Reply:
It is a deployment configuration that is taken from the [maven profiles](https://github.com/EBIvariation/configuration/blob/master/eva-maven-settings.xml#L141)